src/OSPSuite.UI/Views/Importer/ColumnMappingControl.cs

```diff
@@ -68,33 +73,33 @@ public void AttachPresenter(IColumnMappingPresenter presenter)
          _presenter = presenter;
       }
 
-      private RepositoryItemImageComboBox valueRepository(ColumnMappingDTO model)
+      private RepositoryItem descriptionRepository(ColumnMappingDTO model)
       {
-         var repo = new RepositoryItemImageComboBox
+         var descriptionRepository = new UxRepositoryItemComboBox(columnMappingGridView)
          {
             AutoComplete = true,
             AllowNullInput = DefaultBoolean.True,
             CloseUpKey = new KeyShortcut(Keys.Enter)
          };
-         fillComboBoxItems(repo, _presenter.GetAvailableOptionsFor(model));
-         return repo;
+//         descriptionRepository.FillImageComboBoxRepositoryWith(_presenter.GetAvailableOptionsFor(model), x => x.IconIndex);
+         fillComboBoxItems(descriptionRepository, _presenter.GetAvailableOptionsFor(model));
```

Comment:
we have some predefined function fro all of this. However I am not quite sure what the extra stuff is doing. If we want the UI to behave like the rest of the suite, let see if this is needed or not